evalml/tests/pipeline_tests/test_time_series_pipeline.py

```diff
@@ -621,7 +621,7 @@ def test_time_series_pipeline_not_fitted_error(
         X, y = X_y_regression
         clf = time_series_regression_pipeline_class(
             parameters={
-                "Linear Regressor": {"n_jobs": 1},
+                "Random Forest Regressor": {"n_jobs": 1},
```

Comment:
Why change this to RFR?

Reply:
@bchen1116 I can't be 100% sure but due to the occasional issues we're running into with LinearRegressor for time series problems, it might be better to default to a more consistent estimator, but I could be wrong.